test/functional/cmdLineTests/gcsuballoctests/gcsuballoctests.xml

```diff
@@ -88,6 +119,19 @@
   <output regex="no" type="failure">Processing dump event</output>
  </test>
 
+ <test id="-Xgc:suballocatorIncrementSize=0">
+  <command>$EXE$ -Xgc:suballocatorIncrementSize=0 -version</command>
+  <!-- JVMJ9GC036E -Xgc:suballocatorIncrementSize= value must be above 0 -->
+  <output regex="no" type="success">JVMJ9GC036E</output>
+  <!-- JVMJ9VM015W Initialization error for library j9gc26(2): Failed to initialize -->
```

Comment:
This comment is stale: the library is `j9gc29`; could you please fix this and the others?